libs/designsystem/src/lib/components/dropdown/dropdown.component.ts

```diff
@@ -71,6 +81,9 @@ export class DropdownComponent
   @Input()
   placeholder = 'Please select:';
 
+  @Input()
+  cardAlign: cardAlignment = cardAlignment.left;
```

Comment:
Consider naming - would it be better/easier to understand if this was named according to the direction the card/dropdown opens in? E.g. `direction` or `openDirection` or simply `open`, i.e.: `open="left"`
Ionic uses `vertical` and `horizontal` for their FABs: https://ionicframework.com/docs/api/fab#properties
(that API also takes into account to set both the vertical and horizontal direction, whereas only having `direction` would either only support one "compass" direction (NESW) or need to to contain  a matrix of horizontal/vertical aligments if ever needed.... 🤔)
UPDATE: I see now that we already have  `_vertical` and `_horizontal` properties 😂

Reply:
Ha.. I actually though that `_horizontal` was an inherited prop. Totally missed that.
I will re-arrange a bit to not re-invent the wheel too many times

Reply:
refactored